pages/login.page.js

```diff
@@ -0,0 +1,33 @@
+//import $ from 'webdriverio/build/commands/browser/$';
+import BasePage from '../pages/base.page';
+
+class LoginPage extends BasePage {
+
+   //WebElements
+   get usuarioInput(){ return $('[name="username"]') }
```

Comment:
Prefer **ids** whenever possible for locating unique elements. I think in this case, there is an id available for this element.